public/app/plugins/panel/pmm-check/components/Failed/Failed.styles.ts

```diff
@@ -0,0 +1,49 @@
+import { css } from '@emotion/css';
+
+import { GrafanaTheme2 } from '@grafana/data';
+
+export const getStyles = ({ v1: { palette, colors } }: GrafanaTheme2) => ({
+  TooltipWrapper: css`
+    padding: 0.5em;
+  `,
+  TooltipHeader: css`
+    border-bottom: 1px solid #8e8e8e;
```

Comment:
Do we have these colors in theme?

Reply:
Good point. Found the colors from the theme